### DISCUSSION TITLE
Update hacked version

### DISCUSSION THREAD

**Discussion by Mhm00-lgm** [thatskymod/Sky-CotL-Scripts#106] [Q&A] [answered]
The game has been updated, why don't you do anything😿, the action feature doesn't work, the autorun still works, but, some features can't be used. 😞

**Comment by thatskymod** [comment, marked as answer]
Mods in this repository are regularly tested for compatibility. That said, I don't see any apparent issue with the current mods. Be sure to update libLangit to the latest version, [0.4.5](https://github.com/thatskymod/Sky-CotL-Scripts/blob/6a5d6946eb5661e5e14f714d56ca0f4157b3c351/Canvas/Libs/libLangit045.so) and Canvas to [1.5.7](https://github.com/thatskymod/Sky-CotL-Scripts/blob/91728bfbadc25e56bdf01cd8c798b387e2d74014/Canvas/Canvas.apk). Other mods like libsMusic should be removed to prevent Canvas from crashing, but you can keep libRoman to receive the next update once it's ready.
